### ISSUE TITLE
Import from Wallabag v1 - MySQL error 1366 Incorrect string value: '\xF0\x9F\x94\x93\xF0\x9F...' for column 'content' at row 1

### ISSUE THREAD

**Issue by 4oo4** [wallabag/wallabag#2057]
### Issue details

I was attempting to do an import from my Wallabag v1.9.1, and was getting a 500 error when I tried to import through the app itself. When I tried it the console way per [this](https://github.com/wallabag/wallabag/issues/1764#issuecomment-218476406), it would import a chunk of articles and then throw up this:

```
[Doctrine\DBAL\Exception\DriverException]                                                                                                        An exception occurred while executing 'INSERT INTO `wallabag_entry` (title, url, is_archived, is_starred, content, created_at, updated_at, mimetype, language, reading_time, domain_name, preview_picture, is_public, user_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["The Attack That Broke the Dark Web\u2014and How Tor Plans to Fix It", "http:\/\/fusion.net\/story\/238742\/tor-carnegie-mellon-attack\/", 1, "0", "\n\t\t<p>Law enforcement has been complaining for years about the Web \u201cgoing dark,\u201d saying that encryption and privacy tools are frustrating their ability to track criminals online.
```

---Snip Article Contents---

```
  SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x94\x93\xF0\x9F...' for column 'content' at row 1
 [Doctrine\DBAL\Driver\PDOException]
  SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x94\x93\xF0\x9F...' for column 'content' at row [PDOException]
  SQLSTATE[HY000]: General error: 1366 Incorrect string value: '\xF0\x9F\x94\x93\xF0\x9F...' for column 'content' at row 1
```

I had another article cause it to error out like that, I deleted the article in question did another v1 export, and an another clean import in the console, but still got the same message. I tried it with `--ansi` and `--no-ansi`, but still the same result.

The only other thing I can think of doing is keep deleting the articles it throws up, but I'm not sure how many articles I have that are causing this error.
### Environment
- wallabag version (or git revision) that exhibits the issue: v2.0.4
- How did you install wallabag? Via `git clone` or by downloading the package? `git clone`
- Last wallabag version that did not exhibit the issue (if applicable): NA
- php version: 7.0.4-7ubuntu2
- OS: Ubuntu 16.04
- type of hosting (shared or dedicated): dedicated
- which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MySQL 5.7.12-0ubuntu1
### Steps to reproduce/test case

I do not have access at the moment to the json file I was trying to import, but I will post this when I get home so others can try to import it.

EDIT: Here's a download link for the file: [https://drive.google.com/file/d/0B-kDm3f5EOC4b09CdTdMbGFFQXM/view?usp=sharing](https://drive.google.com/file/d/0B-kDm3f5EOC4b09CdTdMbGFFQXM/view?usp=sharing). Apologies for the size (84 MB), I'm a pretty heavy user :)

**Comment by 4oo4** [comment]
I noticed this hasn't gotten much attention yet, can someone point me to where I should be looking in the source to take a stab at this?

**Comment by tcitworld** [comment]
You could try to convert your mysql database to UTF8-MB4 : http://stackoverflow.com/questions/20411440/incorrect-string-value-xf0-x9f-x8e-xb6-xf0-x9f-mysql

**Comment by 4oo4** [comment]
Awesome, I was thinking it was something like that, I will give that a try. Are you meaning my v1 database that I'm importing from, or in my v2 database (or both?)

**Comment by tcitworld** [comment]
I'm guessing both.

**Comment by 4oo4** [comment]
I set the database, the tables, and the string columns as the following, but still got the same error when I tried a clean import.
```
utf8mb4_unicode_ci
utf8mb4_bin
utf8mb4_general_ci
```
I also noticed this time when I import, that the article that keeps tripping it up it has these unicode symbols, not sure if that points to which UTF I need to set them to?
```
\u201d 
\u2019c
\u201c
```

**Comment by 4oo4** [comment]
So, I ended up just going through the articles it threw up one by one and it turned out that there were only 5 of them, so I just ended up manually re-added those. The import worked great, considering that we were importing around 3000 articles and only 5 failed!

I'm thinking this was caused by a weird import job for those articles in Wallabag v1, or something strange with the sites I grabbed them from. Either way, seems to be a MySQL problem rather than anything with Wallabag.
